components/HTTP/src/RESTServer.cpp

```diff
@@ -439,8 +438,10 @@ esp_err_t websocketHandler(httpd_req_t* req)
 void start_server()
 {
     http::HttpServer server_instance = http::HttpServer();
+
+    server_instance.start();
```

Comment:
Could we check for errors here? So `start_server` returns `esp_err_t` on case of failure.